Makefile.am

```diff
@@ -27,7 +30,6 @@ EXTRA_DIST = \
     version.sh
 
 include $(srcdir)/src/Makemodule.am
-include $(srcdir)/src/Makemodule-local.am # Optional project-local hook
```

Comment:
Looks like the version of zproject used did not include the android patches for local make modifications.